src/client-ducky/src/components/Reservation.js

```diff
@@ -26,54 +27,51 @@ const ReservationData = {
 
 const Reservation = ({id, isAuthenticated, token}) => {
 
-    const [resData, setResData] = useState({});
-    const [resID, setResID] = useState(id);
-    const [qrCode, setQrCode] = useState({});
+    const [resData, setResData] = useState("");
 
     useEffect(() => {
-        if (Object.keys(resData).length === 0) {
-            let path = window.location.pathname
-            path = path.split("/");
-            console.log("pathname : ",path[path.length - 1]);
-            if (path[path.length - 2] === "reservation") {
-                let idx = Number(path[path.length - 1]);
-                setResID(idx);
-                console.log("res data", ReservationData[resID]);
-                setResData(ReservationData[resID]);
-                // setQrCode(data); TODO make a api call to /api/qrcode/?data=someURL
-            }
-        }
+        debugger;
+        let paths = window.location.pathname.split("/");
+        let l = paths.length;
+        api.get(`reservations/${paths[l-1]}/`)
```

Comment:
This throws a 401 unauthorized error. But its the same url and token as insomnia which gets the right info